File: AndroidPart/app/src/main/java/com/example/dell/dbtest/Models/StoryModel.java
package com.example.dell.dbtest.Models;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by dell on 2016/11/5.
 */

public class StoryModel implements Serializable
{
    public String story_id;
    public String user_id;
    public String title;
    public String content;
    public String photo;
    public char state;
    // No OnShow End (N O E)
    public char mshow;
    // No Yes (N Y)
    public Date editTime;

}

File: WebService/WebService1/Model/StoryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace STWebService.Model
{
    public class StoryModel
    {
        public string story_id { get; set; }
        public string user_id { get; set; }
        public string title { get; set; }
        public string content { get; set; }
        public Byte[] photo { get; set; }
        public char state { get; set; }
        public char mshow { get; set; }
        public DateTime editTime { get; set; }
    }
}